Makefile

```diff
@@ -96,7 +97,13 @@ lint: $(GOLANGCI_LINT)
 
 fmt:
 	@echo gofmt		# Show progress, real gofmt line is too long
-	find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w
+	@if [ $(OS_NAME) == 'linux' ] ; then \
+		find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w; \
+	fi
+	# BSD (MacOS) xargs does not have option -r (--no-run-if-empty)
+	@if [ $(OS_NAME) = 'darwin' ] ; then \
+		find pkg cmd test -name '*.go' | xargs gofmt -s -l -w; \
```

Comment:
Actually we can just delete the -r, it'll never be empty (or if it is, somebody's deleted the entire code-base, which probably deserves to be an error!) I was coming from writing a different script where the -r was needed and I put it by reflex.

Reply:
thx